static/app/views/eventsV2/table/queryField.tsx

```diff
@@ -332,7 +338,7 @@ class QueryField extends React.Component<Props> {
       }
     }
 
-    if (fieldValue?.kind === 'field') {
+    if (fieldValue?.kind === 'field' || fieldValue.kind === 'calculatedField') {
```

Comment:
```suggestion
    if (fieldValue?.kind === 'field' || fieldValue?.kind === 'calculatedField') {
```
the first part of this condition uses an optional so I think we probably want to use an optional for the second part as well.